developer/tutorial/index.md

```diff
@@ -54,7 +54,7 @@ at some options. When you find one you're happy with (the MIT license is highly
 Gem::Specification.new do |spec|
   spec.name        = 'puppet-lint-trailing_newline-check'
   spec.version     = '1.0.0'
-  spec.homepage    = 'https://github.com/rodjek/puppet-lint-trailing_newline-check'
+  spec.homepage    = 'https://github.com/puppetlabs/puppet-lint-trailing_newline-check'
```

Comment:
In this case it refers to an existing external repo (and many of the same links in this file). I'd not update them.